macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -53,6 +53,8 @@
     'anomaly_direction': 'both',
     'store_result_rows_in_own_table': true,
     'mute_dbt_upgrade_recommendation': false,
+    'calculate_failed_count': true,
+    'use_temp_tables': true
```

Comment:
We should clarify that this is in the context of tests.
Think about how you can convey that in the naming.

Reply:
on it